src/java.base/unix/native/libmallochooks/mallochooks.c

```diff
@@ -0,0 +1,1016 @@
+/*
+ * Copyright (c) 2023 SAP SE. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Don't need to credit Oracle here (and in other new files that you added and that are not substantially copied from something from OpenJDK)